ts/Core/Axis/AxisOptions.d.ts

```diff
@@ -151,6 +151,7 @@ export interface AxisOptions {
     minorTickLength: number;
     minorTickPosition: AxisTickPositionValue;
     minorTicks?: boolean;
+    minorTicksPerMajor?: number;
```

Comment:
Since we're giving this number in the defaults, it doesn't have to be optional. We can remove the question mark.